src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,34 @@
 package mate.academy.lib;
 
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
+import mate.academy.service.ProductService;
+import mate.academy.service.impl.FileReaderServiceImpl;
+import mate.academy.service.impl.ProductParserImpl;
+import mate.academy.service.impl.ProductServiceImpl;
+
 public class Injector {
```

Comment:
Injector is not implemented

Reply:
![image](https://github.com/mate-academy/jv-dependency-injection/assets/114025933/ca5c424d-66bb-40e5-a1ca-8b511617587c)
how is it not implemented?